oop/vue.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/api': {
-        target: 'https://oop-web-a3c641-dev.apps.silver.devops.gov.bc.ca',
+        target: 'https://oop-web-a3c641-test.apps.silver.devops.gov.bc.ca',
```

Comment:
Please change this back to "dev". We still want to use the DEV environment for local development.

Reply:
Updated and pushed